src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,6 +124,12 @@ private int RunExecutable()
                 return result;
             }
 
+            // Add Nuget Packages Probing Path
+            var nugetPackagesRoot = _context.PackagesDirectory;
```

Comment:
I believe the work to make cli a SharedFramework app is underway. 

When doing "dotnet x.dll" it seems like we have two separate scenarios

Scenario 1 maps this to dotnet exec x.dll and shouldn't depend on the nuget cache
Scenario 2 maps this to dotnet run x.dll and will depend on the nuget cache probing path

Is this a correct understanding?